docs/guide/front-end-helpers.md

```diff
@@ -180,7 +182,7 @@ There is also a `runPlayers` method on `apos.util`. That is run for us using `ap
 
 ## HTTP request methods
 
-These are all available in the browser on `apos.util`, e.g., `apos.util.addClass(el, 'is-active')`. They include wrappers for common browser APIs, classic DOM traversal methods, and Apostrophe-specific utilities.
+These are all available in the browser on `apos.http`, e.g., `apos.http.get('/api/v1/article')`. They include for specific HTTP request method utilities as well as other helpers for making HTTP requests.
```

Comment:
"for specific HTTP request method utilities" = grammar issues

How about:

"They include utility methods to make requests with each HTTP method (GET, POST, PATCH, PUT and DELETE), as well as..."

Reply:
like